### PR TITLE
Add additional support for parachutes

### PR DESCRIPTION
**Pull request by amy-chen-skydio**
- Add messages to report parachute status and information
 - Add commands to arm with additional trigger methods and to shutdown the module

### CODE REVIEW

**Review by jenke-skydio — COMMENTED** [review]
lgtm.  Would be good to get another reviewer before approved

**Review by abe-skydio — APPROVED** [review]
lgtm

**Review by ross-skydio — APPROVED** [review]

**Pull request closed without merging**
